### ISSUE TITLE
[Designer]: At 400% zoom, All content present on "Pick a sample as a starting point" window are not visible.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6263]
### Problem Description

[30645314](https://microsoft.visualstudio.com/OS/_workitems/edit/30645314)

At 400% zoom, All content present on "Pick a sample as a starting point" window are not visible.
Note: Same issue is repro for 'Pic2card Dialog for Image Upload' window

### Card JSON

```JSON
No card
```
